Raspberry Pi/RaspberrySide/PythonCV.py

```diff
@@ -47,6 +47,7 @@
         cam.set(cv2.cv.CV_CAP_PROP_BRIGHTNESS, brightness)
         cam.set(cv2.cv.CV_CAP_PROP_SATURATION, saturation)
         cam.set(cv2.cv.CV_CAP_PROP_EXPOSURE, exposure) # not working on the old camera
+        cam.set(cv2.cv.CV_CAP_PROP_BUFFERSIZE, 1)  # Eliminates cv buffer, this way we always recv the latest image
```

Comment:
This should set the memory buffer to a single frame, thus making us read the most current frame at all times.
However, from the documentation: this is only supported by DC1394 v 2.x backend currently - which I don't know what it means or whether we have it or not.
Another suggestion was to run the `read` in `VisionManager` several times sequentially - in order to clear any buffer. This should account for the difference in capture rates. ie: if the camera works at 24 fps, but we only read 4 fps we need to read and throw away 6 frames, and only parse the last one.